ui-tests/src/main/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -290,13 +312,13 @@ public void clickChangeDate (){
 		clickOn(CHANGE_DATE);
 	}
 
-	public void enterDate(){
+	public void enterDate() {
 		Calendar currentDay = Calendar.getInstance();
 	 	Integer dayOfMoth = currentDay.get(Calendar.DAY_OF_MONTH);
-	 	A: for(int j = 6; j > 0; j--) {
+	 	A: for(int j = 6; j > 0; j--) 
 			for (int i = 7; i > 0; i--) {
 				WebElement day = findElement(By.xpath(String.format("//table[@class=' table-condensed']/tbody/tr[%s]/td[%s]",j, i)));
-				if(!day.getAttribute("class").contains("disabled")) {
+				if(!day.getAttribute("class").contains("disabled"))

```

Comment:
Will do! I'll adhere to that convention in the future. Thanks for pointing it out.